docs/src/pages/api-reference/react.mdx

```diff
@@ -407,6 +410,8 @@ const { startUpload, isUploading, permittedFileInfo } = useUploadThing(
 | onUploadProgress       | function                                             | No       | Added in `v5.1` | callback function for upload progress                                                                                       |
 | onBeforeUploadBegin    | `(files: File[]) => File[]`                          | No       | Added in `v5.8` | callback function called before uploading starts. The files returned are the files that will be uploaded                    |
 | onUploadBegin          | function                                             | No       | Added in `v5.4` | callback function for upload begin                                                                                          |
+| onUploadAborted        | function                                             | No       | Added in `v6.7` | callback function when upload is aborted                                                                                    |
```

Comment:
this isn't avail on hook - just components.

callback here would be semver breaking, we need to throw the `startUpload` fn i think?

Reply:
Now that I think about it, it makes sense that we wouldn't handle that in the hook